.gitignore

```diff
@@ -1,2 +1,3 @@
 target
-.env
\ No newline at end of file
+Cargo.lock
```

Comment:
I understand that it should be better to ignore the lock file and I agree to do it. But it should be done on another PR, it is not the goal of this one.

Reply:
Related to #204.